tests/test_requests.py

```diff
@@ -1741,10 +1816,12 @@ def test_json_param_post_should_not_override_data_param(self, httpbin):
         prep = r.prepare()
         assert 'stuff=elixr' == prep.body
 
-    def test_response_iter_lines(self, httpbin):
+    @pytest.mark.parametrize('decode_unicode', (True, False))
+    def test_response_iter_lines(self, httpbin, decode_unicode):
```

Comment:
Nice catch
I overwrote the commit, and fixed the unused `decode_unicode` parameter :ok_hand: